src/main/java/frc/robot/RobotContainer.java

```diff
@@ -352,6 +353,16 @@ public void resetOdometry() {
     commands.add(new TestControlPanel(controlPanelSubsystem).withTimeout(30));
     commands.add(new TestClimb(climbSubsystem).withTimeout(10));
 
+    //add shooter tests
+    commands.add(new TestShooterCommand(shooterSubsystem, 500).withTimeout(2)
+      .andThen(new TestShooterCommand(shooterSubsystem, 1000).withTimeout(2))
+      .andThen(new TestShooterCommand(shooterSubsystem, 2500).withTimeout(2))
+      .andThen(new TestShooterCommand(shooterSubsystem, 2750).withTimeout(2))
+      .andThen(new TestShooterCommand(shooterSubsystem, 3000).withTimeout(2))
+      .andThen(new TestShooterCommand(shooterSubsystem, 3500).withTimeout(2))
+      .andThen(new TestShooterCommand(shooterSubsystem, 4000).withTimeout(2))
```

Comment:
Should the last test turn the shooter off, I think it would be left running after the tests are complete.